backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/exceptions/AdempiereException.java

```diff
@@ -246,6 +247,11 @@ public AdempiereException(final String message)
 		this.mdcContextMap = captureMDCContextMap();
 	}
 
+	public AdempiereException(final String message, final Object... arguments)
```

Comment:
pls don't introduce this method because
* is misleading - u don't know what to expect and how it will be parsed
*  and also facilitates the developer to avoid using ad_messages :)